src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todos } from '../../types/Todos';
+
+interface Props {
+  todo: Todos;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }): JSX.Element => (
+  <article
+    data-id={todo.id}
+    className={todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user && <UserInfo user={todo.user} key={todo.user.id} />}
```

Comment:
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```
you don't need the key prop here